Session/Settings/SettingsViewModel.swift

```diff
@@ -406,7 +424,7 @@ class SettingsViewModel: SessionTableViewModel, NavigationItemSource, Navigatabl
                             ),
                             title: "vc_settings_invite_a_friend_button_title".localized(),
                             onTap: {
-                                let invitation: String = "Hey, I've been using Session to chat with complete privacy and security. Come join me! Download it at https://getsession.org/. My Session ID is \(profile.id) !"
+                                let invitation: String = "Hey, I've been using Session to chat with complete privacy and security. Come join me! Download it at https://getsession.org/. My Session ID is \(state.profile.id) !"
```

Comment:
Mentioned this in an earlier comment but would be nice if this was localised (ignore if solved in Strings project)